modules/apps/frontend-css/frontend-css-variables-web/src/test/resources/com/liferay/frontend/css/variables/web/internal/servlet/taglib/liferay_css_variables_2.html

```diff
@@ -1,13 +1,13 @@
 <style data-senna-track="temporary" type="text/css">
-:root {
---color: red;
-}
-body {
---color: green;
---font: Comic Sans;
-}
-:root {
---color: yellow;
---font: Arial;
-}
+	:root {
+		--color: red;
+	}
+	body {
+		--color: green;
+		--font: Comic Sans;
```

Comment:
I sure would like to see quotes around this, just to show that the system works when the user provides them (ie. would need to add the quotes in the test too).

![just-sayin](https://user-images.githubusercontent.com/7074/85522865-7b206100-b606-11ea-87de-d9f07b3f5cbc.gif)

Reply:
OK, OK.... I'll do it...

![rolling eyes](https://media.giphy.com/media/3oAt2dA6LxMkRrGc0g/giphy.gif)

Reply:
done